kolibri/plugins/learn/assets/src/views/search-page/index.vue

```diff
@@ -160,7 +153,30 @@
       },
       numFiltered() {
         return this.filteredResults.length;
-      }
+      },
+      noResultsMsg() {
```

Comment:
Sorry man. I think I like [this pattern](https://github.com/christianmemije/kolibri/blob/2b143e2d9b066577b038f38f4f5b4a55354b3ed3/kolibri/plugins/learn/assets/src/views/search-page/index.vue#L140) more, in terms of legibility :/